pallets/allocations/src/weights.rs

```diff
@@ -57,106 +57,68 @@ pub trait WeightInfo {
+		Weight::from_ref_time(22_800_000_u64)
+			.saturating_add(T::DbWeight::get().reads(7_u64))
+			.saturating_add(T::DbWeight::get().writes(5_u64))
 	}
-	// Storage: Allocations SessionQuotaRenewSchedule (r:1 w:1)
-	// Storage: Allocations MintCurveStartingBlock (r:1 w:1)
-	// Storage: Allocations NextSessionQuota (r:1 w:0)
-	// Storage: System Number (r:1 w:0)
-	// Storage: System ExecutionPhase (r:1 w:0)
-	// Storage: System EventCount (r:1 w:1)
-	// Storage: System Events (r:1 w:1)
-	// Storage: Allocations SessionQuota (r:0 w:1)
 	fn renew_quota() -> Weight {
-		Weight::from_ref_time(18_960_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(7 as u64))
-			.saturating_add(T::DbWeight::get().writes(5 as u64))
+		Weight::from_ref_time(18_620_000_u64)
+			.saturating_add(T::DbWeight::get().reads(7_u64))
+			.saturating_add(T::DbWeight::get().writes(5_u64))
 	}
-	// Storage: ParachainSystem ValidationData (r:1 w:0)
-	// Storage: Allocations SessionQuotaCalculationSchedule (r:1 w:1)
-	// Storage: Allocations MintCurveStartingBlock (r:1 w:1)
-	// Storage: Balances TotalIssuance (r:1 w:0)
-	// Storage: System Number (r:1 w:0)
-	// Storage: System ExecutionPhase (r:1 w:0)
-	// Storage: System EventCount (r:1 w:1)
-	// Storage: System Events (r:1 w:1)
-	// Storage: Allocations SessionQuotaRenewSchedule (r:1 w:1)
-	// Storage: Allocations SessionQuota (r:0 w:1)
-	// Storage: Allocations NextSessionQuota (r:0 w:1)
 	fn checked_update_session_quota() -> Weight {
-		Weight::from_ref_time(35_240_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(9 as u64))
-			.saturating_add(T::DbWeight::get().writes(7 as u64))
+		Weight::from_ref_time(34_490_000_u64)
+			.saturating_add(T::DbWeight::get().reads(9_u64))
+			.saturating_add(T::DbWeight::get().writes(7_u64))
 	}
-	// Storage: ParachainSystem ValidationData (r:1 w:0)
-	// Storage: Allocations MintCurveStartingBlock (r:0 w:1)
-	// Storage: Allocations SessionQuotaCalculationSchedule (r:0 w:1)
-	// Storage: Allocations SessionQuotaRenewSchedule (r:0 w:1)
 	fn set_curve_starting_block() -> Weight {
-		Weight::from_ref_time(11_690_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(1 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(11_220_000_u64)
+			.saturating_add(T::DbWeight::get().reads(1_u64))
+			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	fn allocate(b: u32) -> Weight {
-		Weight::from_ref_time(73_910_000 as u64)
-			// Standard Error: 9_000
-			.saturating_add(Weight::from_ref_time(26_411_000).saturating_mul(b as u64))
-			.saturating_add(RocksDbWeight::get().reads(8 as u64))
-			.saturating_add(RocksDbWeight::get().reads((1 as u64).saturating_mul(b as u64)))
-			.saturating_add(RocksDbWeight::get().writes(6 as u64))
-			.saturating_add(RocksDbWeight::get().writes((1 as u64).saturating_mul(b as u64)))
+		Weight::from_ref_time(0_u64)
```

Comment:
again 0

Reply:
Again a big standard error may be a result of less than enough steps/iterations in the benchmark command.